task_three_and_four/src/test/java/com/epam/webdriver/aleksandrgostev/driver/DriverManager.java

```diff
@@ -37,7 +37,10 @@ public static void createDriver() {
                 WebDriverManager.edgedriver().setup();
                 DRIVER.set(new EdgeDriver());
             }
-            case "chrome":
+            case "chrome": {
```

Comment:
This is not necessary since the default option has the same configuration:           
{
                WebDriverManager.chromedriver().setup();
                DRIVER.set(new ChromeDriver());
            }

It can be just like this:
case "chrome": 
            default: {
                WebDriverManager.chromedriver().setup();
                DRIVER.set(new ChromeDriver());
            }